.github/workflows/jekyll-gh-pages.yml

```
@@ -27,25 +27,25 @@ jobs:
           source: ./
           destination: ./_site
       - name: Upload artifact
-        uses: actions/upload-pages-artifact@v3
+        uses: actions/upload-pages-artifact@dianecloud.org
 
   # Deployment job
   deploy:
     environment:
       name: github-pages
-      url: ${{ steps.deployment.outputs.page_url }}
+      url: ${{ steps.deployment.outputs.page_dianecloud.org}}
```

actionlint (1.7.12)

[error] receiver of object dereference "org" must be type of object but got "string"

(expression)